### PR TITLE
B 21542 int - DoD ID Does Not Populate in Services Counselor Queue

### PR DESCRIPTION
**Pull request by JonSpight**
## Summary
When office users login to MM, their queue is not showing the DOD ID of the customer.

### How to test
1. Login as a customer and create a move.
2. Login as a  Services Counselor verify the DoD ID value is displayed in the counseling Queue
3. submit the move
4. Login as TOO,  verify the DoD ID value is displayed In the task order queue
5. approve the move and create a payment request
6. Login as a or TIO verify the DoD ID value is displayed In the payment request queues

### CODE REVIEW

File: src/pages/Office/HeadquartersQueues/HeadquartersQueues.test.jsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ const GetMountedComponent = (queueTypeToMount) => {
   return wrapper;
 };
 
-const SEARCH_OPTIONS = ['Move Code', 'DoD ID', 'Customer Name', 'Payment Request Number'];
+const SEARCH_OPTIONS = ['Move Code', 'edpi', 'Customer Name', 'Payment Request Number'];
```
Comment (danieljordan-caci) [inline src/pages/Office/HeadquartersQueues/HeadquartersQueues.test.jsx:182]: Typo in `edpi` - should be `edipi`

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

**Review by danieljordan-caci — APPROVED** [review]
As the joker that did the BL item that jacked this up - thanks for the fix! LGTM.

**Comment by pambecker** [comment]
I am not seeing the DoDID being populated in the TIO (Payment request queue).
And all 4 queues are breaking on sort and filter not working on any queues.

**Comment by danieljordan-caci** [comment]
> I am not seeing the DoDID being populated in the TIO (Payment request queue). And all 4 queues are breaking on sort and filter not working on any queues.

Oh nice - let me re-review. Didn't think about checking the TIO queue.

**Comment by JonSpight** [comment]
> > I am not seeing the DoDID being populated in the TIO (Payment request queue). And all 4 queues are breaking on sort and filter not working on any queues.
> 
> Oh nice - let me re-review. Didn't think about checking the TIO queue.

Good catch, I'll look into the filtering/sorting issue. But I do see the DoDID being populated for the TIO.

**Review by loganwc — APPROVED** [review]

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]
Sort isn't working - getting the oops page

Screenshot 2024-11-07 at 11 08 01 AM

```
Operation queues.getServicesCounselingQueue failed: Error: Unprocessable Entity (422)
```

**Comment by danieljordan-caci** [comment]
Is this missing value part of your work? Should be showing up in this list. There's a chance it's a small fix and would be convenient to wrap into this BL item.

Screenshot 2024-11-07 at 11 20 56 AM

**Comment by danieljordan-caci** [comment]
> Sort isn't working - getting the oops page
> 
> Screenshot 2024-11-07 at 11 08 01 AM
> 
> ```
> Operation queues.getServicesCounselingQueue failed: Error: Unprocessable Entity (422)
> ```

I would say so because this AC:
```
As a Services Counselor, TOO, or TIO working a record in my queue,
I want to see the DoD ID of the customer,
So that I am able to identify my customer. 
```

**Comment by pambecker** [comment]
> Is this missing value part of your work? Should be showing up in this list. There's a chance it's a small fix and would be convenient to wrap into this BL item.
> 
> Screenshot 2024-11-07 at 11 20 56 AM

good catch, I agree, this should be updated as well.

**Comment by JonSpight** [comment]
> > Is this missing value part of your work? Should be showing up in this list. There's a chance it's a small fix and would be convenient to wrap into this BL item.
> > Screenshot 2024-11-07 at 11 20 56 AM
> 
> good catch, I agree, this should be updated as well.

Made some changes, Updated the customer info page, The columns should work and are sortable/filterable, you can also search for a move using the DodID as well.

**Review by pambecker — APPROVED** [review]

**Review by danieljordan-caci — APPROVED** [review]
Very nice! LGTM.

**Pull request merged**